.github/boring-cyborg.yml

```diff
@@ -0,0 +1,35 @@
+# Details: https://github.com/kaxil/boring-cyborg
+
+labelPRBasedOnFilePath:
+  area/provider/alibaba:
+    - diagrams/alibabacloud/*
+    - resources/alibabacloud/**/*
+
+  area/provider/aws:
+    - diagrams/aws/*
+    - resources/aws/**/*
+ 
+  area/provider/azure:
+    - diagrams/azure/*
+    - resources/azure/**/*
+ 
+  area/provider/firebase:
+    - diagrams/firebase/*
+    - resources/firebase/**/*
+    
+  # @TODO: extend with other labels
+
+firstPRWelcomeComment: >
+  Congratulations on your first Pull Request!
+  If you have any issues or are unsure about any anything please check our
+  Contribution Guide (https://github.com/mingrammer/diagrams/blob/master/CONTRIBUTING.rst)
```

Comment:
Our contribution guide link is https://github.com/mingrammer/diagrams/blob/master/CONTRIBUTING.md